conf.d/main

```diff
@@ -39,9 +39,15 @@ ghost install --url=http://localhost:2368 --db=mysql --dbhost=localhost --dbuser
 
 sed -i 's/tkldev/ghost/' /etc/hosts
 
+rm /etc/nginx/sites-enabled/default
```

Comment:
This path has changed, but that's my fault (because I didn't complete the nginx changes in a timely manner). So I'll take care of this. Again, I'll take care of it...